python/TestHarness/testers/SchemaDiff.py

```diff
@@ -0,0 +1,165 @@
+#* This file is part of the MOOSE framework
+#* https://www.mooseframework.org
+#*
+#* All rights reserved, see COPYRIGHT for full restrictions
+#* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+#*
+#* Licensed under LGPL 2.1, please see LICENSE for details
+#* https://www.gnu.org/licenses/lgpl-2.1.html
+
+from RunApp import RunApp
+from TestHarness import util
+import os
+
+class SchemaDiff(RunApp):
```

Comment:
Need to add this object as not select-able to the registry

Reply:
Not sure how to do that, or what the registry even is in the context

Reply:
https://github.com/idaholab/moose/blob/a736fd6cb26e9211fd6b6bbb6c9d4fc63acde894/python/TestHarness/TestHarness.py#L457

This is where the testers are loaded; we should skip `SchemaDiff` here